content/javascript/concepts/setTimeout/setTimeout.md

```diff
@@ -0,0 +1,81 @@
+---
+Title: 'setTimeout()'
+Description: 'Allows users to execute callbacks after a period of time expressed in milliseconds'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Parameters'
+  - 'Arguments'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**`setTimeout()`** is a function serviced globally by the window object provided by the user's browser.
+
+## Syntax
+
+**`setTimeout()`** is capable of receiving multiple parameters where the first is a callback function, the second parameter receives a number that represents the time in milliseconds **(1s = 1000ms)**, that defines the time needed for the callback to execute, the third parameter onwards will be the parameters that the callback function would take in case arguments are defined within the callback.
+
+The provided sample code contains some examples of how `setTimeout()` is composed:
+
+```pseudo
+setTimeout(callback)
+
+setTimeout(callback, delay)
+
+setTimeout(callback, delay, param)
+```
+
+## Examples
+
+The following pieces of code will show some examples about the `setTimeout()` implementation:
+
+```shell
+// Defining the function
+function sayHello() {
+  console.log('Hello from Codecademy');
+}
+
+// Calling the function using setTimeout.
+setTimeout(sayHello, 4000);
+```
+
+The output will be displayed after 4 seconds:
+
+```output
+Hello from Codecademy
+```
+
+## Example
+
+The following example can be done by using arrow functions, in which case, instead of declaring it using the word `function`, we can just create it by using `() =>`, for more information visit [arrow functions](https://www.codecademy.com/resources/docs/javascript/arrow-functions).
```

Comment:
```suggestion
The following example shows the use of [arrow functions](https://www.codecademy.com/resources/docs/javascript/arrow-functions). Instead of declaring it using the word `function`, we can just create it by using `() =>`.
```
Such references work better in-line.